runelite-client/src/main/java/net/runelite/client/plugins/antidrag/AntiDragPlugin.java

```diff
@@ -48,14 +48,15 @@
 
 @PluginDescriptor(
 	name = "Anti Drag",
-	enabledByDefault = false
+	enabledByDefault = false,
+	description = "Prevent dragging an item for a specified delay",
+	tags = {"antidrag", "delay", "inventory", "items"}
 )
-@Slf4j
 public class AntiDragPlugin extends Plugin implements KeyListener
 {
 	static final String CONFIG_GROUP = "antiDrag";
 
-	static final int DEFAULT_DELAY = 5;
+	private final int DEFAULT_DELAY = 5;
```

Comment:
This can remain `static` in addition to being private.
```suggestion
	private static final int DEFAULT_DELAY = 5;
```